modules/api/src/main/java/net/countercraft/movecraft/utils/Pair.java

```diff
@@ -0,0 +1,54 @@
+package net.countercraft.movecraft.utils;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.UUID;
+
+public class Pair<L,R> implements Serializable, Map.Entry<L,R>{
```

Comment:
Do not implement your own Pair class. You've made several mistakes in this one, such as breaking the `HashCode` contract.